utils/tf.py

```diff
@@ -43,13 +47,14 @@ def __init__(self, path_to_model: str, output_names: list):
         except AttributeError:
             advertise_aio("TensorFlow")
 
-        self.__graph = self.__initialize_graph(path_to_model)
+        self.graph = self.__initialize_graph(path_to_model, use_opsgraph)
+        self.config = self.__create_config(bench_utils.get_intra_op_parallelism_threads())
```

Comment:
what's the purpose of introducing self.config variable? I dont see it accessed at any later time?

Reply:
it is used in widedeep/run.py in line 52. The reason for this is that sometimes model is used for preprocessing and thus I needed config functionality in the utils/recommendation/widedeep.py file.